### ISSUE TITLE
autocmd event for tabs being reordered (:tabmove etc)

### ISSUE THREAD

**Issue by realh** [neovim/neovim#6916]
I want my GUI to handle tabs instead of letting nvim manage them in text mode. I can keep my GUI in sync with vim using autocmd events TabEnter, TabNew and TabClosed, but I can't find a way to get notified if the tabs are reordered with something like :tabmove. Is there an existing event I've overlooked, or some sort of workaround that doesn't involve spamming the RPC with tab order queries?

FWIW I'm still working with neovim 0.1.7 until Debian catches up with the latest release, but, looking at the docs, I don't think 0.2.x has added any related events.

**Comment by justinmk** [comment]
Thanks for the request. The tabline has already been externalized so GUIs can handle it. Though there is more work to be done.

**Comment by Iron-E** [comment]
Is there any chance this might get implemented as an autocommand? I've linked this issue on another repo which has been redrawing the tabline based on autocommands, and `TabMove` is the only one we're missing so far.

I looked at the docs and it appears we'd have to restructure the plugin to use `nvim_ui_attach()` to make use of the `ext_tabline` events, and I'm not sure that this is actually what we want to do.

Thoughts?
